src/renderer/external/sphero.js

```diff
@@ -45,6 +45,7 @@ cwc.renderer.external.Sphero.prototype.init = function() {
   let rendererInstance = this.helper.getInstance('renderer', true);
   let renderer = this.render.bind(this);
   rendererInstance.setRenderer(renderer);
+  rendererInstance.setServerMode(true);
```

Comment:
This will never work, because the robots are execute inside an sandbox and if you load the sandbox into the local webserver you will get into cross domain issues. Please remove this line.